jsk_kinova_robot/jsk_kinova.rosinstall

```diff
@@ -0,0 +1,11 @@
+# ros_kortex is updating to v2.3.X in master branch
```

Comment:
should we use 'kinova.rosinstall', instead of 'jsk_kinova.rosinstall'? 
https://github.com/jsk-ros-pkg/jsk_robot/pull/1372/files